.travis.yml

```diff
@@ -3,21 +3,19 @@ services:
 - docker
 env:
   global:
-  - DIST_REPO="f5-sphinx-theme"
-  - USER_REPO=$(echo $TRAVIS_REPO_SLUG | cut -f -1 -d / )
-  - S3_DIST_URL="http://d3s2z99mfhr69y.cloudfront.net"
-  - BUILD_HTML_DIR=${TRAVIS_BUILD_DIR}/test-deploy/docs/_build/html
-  - BRANCH_DIR=${DIST_REPO}/travis-builds/${USER_REPO}/${TRAVIS_BRANCH}/
   - AWS_S3_BUCKET=${ARTIFACTS_BUCKET}
-  - UPLOAD_DIR="f5-sphinx-theme/$(date +%F)/${TRAVIS_BUILD_NUMBER}"
+  - USER_REPO=$(echo $TRAVIS_REPO_SLUG | cut -f -1 -d / )
+  - PROJECT_UPLOAD_ROOT="under-review/f5-sphinx-theme"
+  - BUILD_UPLOAD_ROOT="${PROJECT_UPLOAD_ROOT}/${USER_REPO}/${TRAVIS_BRANCH}"
+  - UPLOAD_DIR="${BUILD_UPLOAD_ROOT}/$(date +%F)-${TRAVIS_BUILD_NUMBER}"
 
 language: python
 python:
 - '2.7'
 before_install:
 - git config --global user.email "OpenStack_TravisCI@f5.com"
 - git config --global user.name "Travis F5"
-- docker pull f5devcentral/containthedocs
```

Comment:
I set up the project to use the docker container to build and deploy the test documentation because that's how all of our actual documentation projects do this.